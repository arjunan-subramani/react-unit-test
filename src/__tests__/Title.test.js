import React from 'react';
import renderer from 'react-test-renderer';
import Title from '../Title';

describe("Title Component", () => {
  it('should render the component correctly', () => {
    const tree = renderer.create(<Title text="Employees Information" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
