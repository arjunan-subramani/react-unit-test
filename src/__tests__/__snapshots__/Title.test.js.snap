// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component should render the component correctly 1`] = `
<h1
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  Employees Information
</h1>
`;
